docs/howto/add_pagination_links.md

```diff
@@ -2,47 +2,34 @@
 
 # How to add pagination links
 
-### JSON API adapter
+## JSON API adapter
 
-Pagination links will be included in your response automatically as long as
-the resource is paginated and if you are using the ```JsonApi``` adapter.
+When using the `JsonApi` adapter, pagination links will be automatically included if you use [Kaminari](https://github.com/amatsuda/kaminari)
+or [WillPaginate](https://github.com/mislav/will_paginate) within a Rails controller:
 
-If you want pagination links in your response, use [Kaminari](https://github.com/amatsuda/kaminari)
-or [WillPaginate](https://github.com/mislav/will_paginate).
-
-Although the others adapters does not have this feature, it is possible to
-implement pagination links to `JSON` adapter. For more information about it,
-please see in our docs 
-
-###### Kaminari examples
+* Using Kaminari:
 
 ```ruby
-#array
-@posts = Kaminari.paginate_array([1, 2, 3]).page(3).per(1)
-render json: @posts
-
-#active_record
-@posts = Post.page(3).per(1)
-render json: @posts
+class PostsController < ApplicationController
+  def index
+    posts = Post.page(page).per(per_page)
+    render json: posts
+  end
+end
 ```
 
-###### WillPaginate examples
-
-```ruby
-#array
-@posts = [1,2,3].paginate(page: 3, per_page: 1)
-render json: @posts
-
-#active_record
-@posts = Post.page(3).per_page(1)
-render json: @posts
-```
+* Using WillPaginate:
 
 ```ruby
-ActiveModelSerializers.config.adapter = :json_api
+class PostsController < ApplicationController
+  def index
+    posts = Post.page(page).per_page(per_page)
```

Comment:
yeah, I think the params would be helpful. Otherwise it might lead the reader to believe that we are monkey patching those methods in